src/layouts/Footer/components/Footer-Container/FooterContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+import NavLink from './components/NavLink/NavLink';
```

Comment:
```suggestion
import NavLink from '@/components/NavLink';
```
加上index 索引文件 就可以不用路径里面写两遍

Reply:
Resolved